### PR TITLE
sig multicluster 2021 body

### PR DESCRIPTION
**Pull request by pmorie**
Adds body to 2021 report

### CODE REVIEW

File: sig-multicluster/annual-report-2021.md [hunk 1]
```diff
@@ -3,135 +3,82 @@
...
-   -
+   - Zoom attendance
+   - Youtube meetup views
+   - Complaints about late youtube uploads of meetups :)
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - We welcome all to our meetups to present ideas, ask questions. We're actively looking for help on a few initiatives
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2021.md:37]: Do you happen to have any issues or docs that could be linked here? When we do our large report out, anything that we can use to direct folk is super useful 🙏

File: sig-multicluster/annual-report-2021.md [hunk 2]
```diff
@@ -3,135 +3,82 @@
...
-   -
+   - yes
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - We need more participation, and exchange of ideas from the K8s community.
+   - Anyone who wants to contribute can immediately start picking up our backlog and start proposing designs.
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2021.md:52]: Can you link to your backlog/designs here?

File: sig-multicluster/annual-report-2021.md [hunk 3]
```diff
@@ -3,135 +3,82 @@
...
--
-
-Retired in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
-
-Continuing:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+- ruler
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2021.md:65]: You do have some subprojects :P

https://github.com/kubernetes/community/tree/master/sig-multicluster#subprojects

File: sig-multicluster/annual-report-2021.md [hunk 4]
```diff
@@ -3,135 +3,82 @@
...
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - All reviews and approvers must pay the very very fair entrance fee
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - yes
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2021.md:46]: devstats seems to be having some issues right now, but when its back I can get you a link to insert here that will list all the different vendors 👍

File: sig-multicluster/annual-report-2021.md [hunk 5]
```diff
@@ -3,135 +3,82 @@
...
+- ruler
 
 ## Operational
 
 Operational tasks in [sig-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [X] [README.md] reviewed for accuracy and updated if needed
+- [X] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2021.md:72]: CONTRIBUTING.md is a bit out of date..it still references design proposals in the k/community repo. Not necessarily a blocker to get this merged, but I'd put it on the list of things to update.

File: sig-multicluster/annual-report-2021.md [hunk 6]
```diff
@@ -3,135 +3,82 @@
...
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - We welcome all to our meetups to present ideas, ask questions. We're actively looking for help on a few initiatives
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - All reviews and approvers must pay the very very fair entrance fee
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2021.md:42]: I appreciate the humor ;) but probably not for this doc.  Is there any skillsets or backgrounds that would help contributors if they wanted to start work in multicluster? like if you were writing a job description for someone to hire, is there anything you'd look for?

**Review by mrbobbytables — COMMENTED** [review]
Thanks for getting this going :) Just a few comments

File: sig-multicluster/annual-report-2021.md [hunk 7]
```diff
@@ -3,135 +3,82 @@
...
+   - Multicluster Services
+   - ClusterProperty
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
-
-   -
-   -
-   -
+   - Multicluster leader election has been discussed several times but is not currently associated with any KEP
+   - Multicluster network policy (driven by SIG-Net)
```
Comment (JeremyOT) [inline sig-multicluster/annual-report-2021.md:11]:
```suggestion
   - Multicluster network policy (driven by SIG-Net) [Extending KEP-2091](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2091-admin-network-policy)
```

File: sig-multicluster/annual-report-2021.md [hunk 8]
```diff
@@ -3,135 +3,82 @@
...
+   - Multi-network as potential MCS extension
```
Comment (JeremyOT) [inline sig-multicluster/annual-report-2021.md:12]:
```suggestion
   - Multi-network as potential MCS extension. [thoughs](https://docs.google.com/document/d/1IueF8FyGKKRPcg8Zvua84FDC327J3UdNiTMO5W7s7tQ/edit?usp=sharing)
```

File: sig-multicluster/annual-report-2021.md [hunk 9]
```diff
@@ -3,135 +3,82 @@
...
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
+   
+   - Work API: progress stalled out, we should make progress or consider archival
```
Comment (JeremyOT) [inline sig-multicluster/annual-report-2021.md:25]:
```suggestion
   - [Work API](https://github.com/kubernetes-sigs/work-api): progress stalled out, we should make progress or consider archival
```

**Review by JeremyOT — COMMENTED** [review]
Added a few changes with links, I'll finish addressing comments later today/this evening

**Pull request closed without merging**